Source/DafnyLanguageServer.Test/Various/ShutdownTest.cs

```diff
@@ -71,7 +71,7 @@ public class ShutdownTest {
 
       // Wait for the language server to kill itself by waiting until it closes the output stream.
       await process.StandardOutput.ReadToEndAsync();
-      Thread.Sleep(300); // Give the process some time to die
+      await Task.Delay(400); // Give the process some time to die
```

Comment:
Why not `WaitForExitAsync`?

Reply:
I don't know what this does but sometimes a test fail because the process did not die.
@keyboardDrummer ?